airflow/www/views.py

```diff
@@ -1654,30 +1655,37 @@ def index(self):
         session.expunge_all()
         session.commit()
         session.close()
-        dags = dagbag.dags.values()
-        if do_filter:
-            if owner_mode == 'ldapgroup':
-                dags = {
-                    dag.dag_id: dag
-                    for dag in dags
-                    if (
-                        dag.owner in current_user.ldap_groups and (not dag.parent_dag)
-                    )
-                }
-            elif owner_mode == 'user':
-                dags = {
-                    dag.dag_id: dag
-                    for dag in dags
-                    if (
-                        dag.owner == current_user.user.username and (not dag.parent_dag)
-                    )
-                }
+
+        # read webserver_dags from local dagbag
+
+        webserver_dags = dagbag.dags.values()
+        # remove subdags
+        webserver_dags = [dag for dag in webserver_dags if not dag.parent_dag]
```

Comment:
Combine the two lines above, and I would also name it unfiltered_webserver_dags to differentiate it from the final state that the webserver_dags variable tasks on.